### ISSUE TITLE
About fine-tuning resnext-101 on hmdb51

### ISSUE THREAD

**Issue by wangs311** [kenshohara/3D-ResNets-PyTorch#170]
Hi.
I want to fine-tune resnext-101 on hmdb51_split1 with lr=0.001 and weight_decay=1e-5,but after 200 epoch, the val only get 10% accuracy. I have tested the supplied model resnext-101-64f-kinetics-hmdb51_split1,which can get 69% accuracy.  What do i miss? Thank you!
Here is my opt.json
{
    "root_path": "",
    "video_path": "=/data/hmdb51",
    "annotation_path": "/data/hmdb51_1.json",
    "result_path": "results",
    "dataset": "hmdb51",
    "n_classes": 400,
    "n_finetune_classes": 51,
    "sample_size": 112,
    "sample_duration": 16,
    "initial_scale": 1.0,
    "n_scales": 5,
    "scale_step": 0.84089641525,
    "train_crop": "corner",
    "learning_rate": 0.001,
    "momentum": 0.9,
    "dampening": 0.9,
    "weight_decay": 1e-05,
    "mean_dataset": "activitynet",
    "no_mean_norm": false,
    "std_norm": false,
    "nesterov": false,
    "optimizer": "sgd",
    "lr_patience": 10,
    "batch_size": 16,
    "n_epochs": 300,
    "begin_epoch": 1,
    "n_val_samples": 3,
    "resume_path": "",
    "pretrain_path": "models/resnext-101-64f-kinetics.pth",
    "ft_begin_index": 0,
    "no_train": false,
    "no_val": false,
    "test": false,
    "test_subset": "val",
    "scale_in_test": 1.0,
    "crop_position_in_test": "c",
    "no_softmax_in_test": false,
    "no_cuda": false,
    "n_threads": 4,
    "checkpoint": 10,
    "no_hflip": false,
    "norm_value": 1,
    "model": "resnext",
    "model_depth": 101,
    "resnet_shortcut": "B",
    "wide_resnet_k": 2,
    "resnext_cardinality": 32,
    "manual_seed": 1,
    "scales": [
        1.0,
        0.84089641525,
        0.7071067811803005,
        0.5946035574934808,
        0.4999999999911653
    ],
    "arch": "resnext-101",
    "mean": [
        114.7748,
        107.7354,
        99.475
    ],
    "std": [
        38.7568578,
        37.88248729,
        40.02898126
    ]
}

**Comment by wangs311** [comment]
I have set --ft_begin_index 4 . The accuracy is 16% now. How to raise  accuracy?

**Comment by slighting666** [comment]
There is no   @@resnet34,50,152  in the link 
Do you have a pretraining model tuned in ucf101 or hmdb51? I need it very much, thank you.
@wangs311

**Comment by slighting666** [comment]
There is no resnet34,50  in the link 
Do you have a pretraining model tuned in ucf101 or hmdb51? I need it very much, thank you. @wangs311

**Comment by wangs311** [comment]
hi, the resnet34,50&nbsp;pre-trained models&nbsp;on Kinetics&nbsp;are available in the link.
You can tuned in ucf101 or hmdb51 base it.
You can consider these parameters:


python main.py --root_path ~/data --video_path hmdb51_videos/jpg --annotation_path hmdb51.json \
--result_path results --dataset hmdb51 --n_classes 400 --n_finetune_classes 51 \
--pretrain_path models/resnet-34-kinetics.pth --ft_begin_index 4 \ --model resnet --model_depth 34 --resnet_shortcut A --batch_size 128 --n_threads 4 --checkpoint 5


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"slighting666"<notifications@github.com&gt;;
发送时间:&nbsp;2020年2月26日(星期三) 下午3:12
收件人:&nbsp;"kenshohara/3D-ResNets-PyTorch"<3D-ResNets-PyTorch@noreply.github.com&gt;;
抄送:&nbsp;" "<1216417625@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [kenshohara/3D-ResNets-PyTorch] About fine-tuning resnext-101 on hmdb51 (#170)




There is no resnet34,50  in the link
 Do you have a pretraining model tuned in ucf101 or hmdb51? I need it very much, thank you. @wangs311

**Comment by wangs311** [comment]
hi, the resnet34,50 pre-trained models on Kinetics are available in the link.
You can tuned in ucf101 or hmdb51 base it.
You can consider these parameters:

python main.py --root_path ~/data --video_path hmdb51_videos/jpg --annotation_path hmdb51.json \
--result_path results --dataset hmdb51 --n_classes 400 --n_finetune_classes 51 \
--pretrain_path models/resnet-34-kinetics.pth --ft_begin_index 4 \
--model resnet --model_depth 34 --resnet_shortcut A --batch_size 128 --n_threads 4 --checkpoint 5

**Comment by slighting666** [comment]
It seems that the link has been deleted. Could you send it to me? My qq1149745774


---Original---
